README.md

```diff
@@ -100,6 +100,7 @@ Below are links to profiles where you can see Readme Typing SVGs in action!
 [![HauseMasterZ](https://github.com/HauseMasterZ.png?size=60)](https://github.com/HauseMasterZ "HauseMaster on Github")
 [![hyskoniho](https://github.com/hyskoniho.png?size=60)](https://github.com/hyskoniho "hyskoniho on Github")
 [![elvisisvan](https://github.com/elvisisvan.png?size=60)](https://github.com/elvisisvan "elvisisvan on Github")
+[![elvisisvan](https://github.com/elvisisvan.png?size=60)](https://github.com/Nquenan "Nquenan on Github")
```

Comment:
Make sure the alt text and image are updated too, feel free to commit this suggestion with the button

```suggestion
[![Nquenan](https://github.com/Nquenan.png?size=60)](https://github.com/Nquenan "Nquenan on Github")
```

Reply:
Ofc! Sorry for that ty!!